porch/controllers/packagevariants/pkg/controllers/packagevariant/packagevariant_controller.go

```diff
@@ -722,27 +722,13 @@ func ensurePackageContext(pv *api.PackageVariant,
 		return nil
 	}
 
-	// find the kptfile.kpt.dev ConfigMap, it must be in package-context.yaml
```

Comment:
Are the changes in this file related to adding the go mod?

Reply:
Some of them, because the fn lib was updated. I pulled this set of change from a different PR.